packages/integration-tests/fixtures/eslint-v8/package.json

```diff
@@ -1,5 +1,5 @@
 {
   "devDependencies": {
-    "eslint": "8.57.0"
+    "eslint": "9.0.0"
```

Comment:
The fixture is called "v8" so we should leave this at v8.
Its a regression test to try and catch some basic backwards incompatibility failures.